OSVRUnreal/Plugins/OSVR/Source/OSVR/Private/OSVRCustomPresent.h

```diff
@@ -280,15 +289,34 @@ class FOSVRCustomPresent : public FRHICustomPresent
 
     virtual OSVR_Pose3 GetHeadPoseFromCachedRenderInfoCollectionImpl(OSVR_RenderInfoCollection renderInfoCollection, OSVR_RenderInfoCount index) = 0;
 
-    virtual void UpdateCachedRenderInfoCollection(OSVR_RenderInfoCollection &renderInfoCollection)
+    virtual bool UpdateCachedRenderInfoCollection(OSVR_RenderInfoCollection &renderInfoCollection)
     {
         OSVR_ReturnCode rc;
+        if (!mRenderManager)
+        {
+            UE_LOG(FOSVRCustomPresentLog, Warning,
+                TEXT("OSVRCustomPresent::UpdateCachedRenderInfoCollection: mRenderManager is not yet initialized."));
+            return false;
+        }
+
         if (renderInfoCollection) {
             rc = osvrRenderManagerReleaseRenderInfoCollection(renderInfoCollection);
-            check(rc == OSVR_RETURN_SUCCESS);
+            if (rc != OSVR_RETURN_SUCCESS)
+            {
+                UE_LOG(FOSVRCustomPresentLog, Warning,
+                    TEXT("OSVRCustomPresent::UpdateCachedRenderInfoCollection: osvrRenderManagerReleaseRenderInfoCollection call failed."));
+                return false;
+            }
         }
         rc = osvrRenderManagerGetRenderInfoCollection(mRenderManager, mRenderParams, &renderInfoCollection);
-        check(rc == OSVR_RETURN_SUCCESS);
+        if (rc != OSVR_RETURN_SUCCESS)
+        {
+            UE_LOG(FOSVRCustomPresentLog, Warning,
```

Comment:
I could probably limit it, but if they're getting it every frame something more serious is wrong instead of just a transient error.